app/frontend/models/jurisdiction.ts

```diff
@@ -1,76 +1,43 @@
 import { t } from "i18next"
-import { Instance, cast, flow, toGenerator, types } from "mobx-state-tree"
+import { Instance, types } from "mobx-state-tree"
 import { Descendant } from "slate"
-import { createSearchModel } from "../lib/create-search-model"
 import { withEnvironment } from "../lib/with-environment"
-import { withRootStore } from "../lib/with-root-store"
 import { EUserSortFields } from "../types/enums"
-import { IContact, ISort } from "../types/types"
+import { IContact } from "../types/types"
 import { toCamelCase } from "../utils/utility-funcitons"
 import { UserModel } from "./user"
 
 export const JurisdictionModel = types
-  .compose(
-    types.model("JurisdictionModel", {
-      id: types.identifier,
-      name: types.string,
-      qualifiedName: types.string,
-      reverseQualifiedName: types.string,
-      localityType: types.string,
-      qualifier: types.string,
-      reviewManagersSize: types.number,
-      reviewersSize: types.number,
-      permitApplicationsSize: types.number,
-      description: types.maybeNull(types.string),
-      // JSONB data type can be represented as a frozen type
-      checklistSlateData: types.maybeNull(types.frozen<Descendant[]>()),
-      lookOutSlateData: types.maybeNull(types.frozen<Descendant[]>()),
-      contacts: types.array(types.frozen<IContact>()),
-      createdAt: types.Date,
-      updatedAt: types.Date,
-      tableUsers: types.array(types.safeReference(UserModel)),
-      query: types.maybeNull(types.string),
-      sort: types.maybeNull(types.frozen<ISort<EUserSortFields>>()),
-      currentPage: types.optional(types.number, 1),
-      totalPages: types.maybeNull(types.number),
-      totalCount: types.maybeNull(types.number),
-      countPerPage: types.optional(types.number, 10),
-    }),
-    createSearchModel<EUserSortFields>("fetchUsers")
-  )
+  .model("JurisdictionModel", {
+    id: types.identifier,
+    name: types.string,
+    qualifiedName: types.string,
+    reverseQualifiedName: types.string,
+    localityType: types.string,
+    qualifier: types.string,
+    reviewManagersSize: types.number,
+    reviewersSize: types.number,
+    permitApplicationsSize: types.number,
+    description: types.maybeNull(types.string),
+    // JSONB data type can be represented as a frozen type
+    checklistSlateData: types.maybeNull(types.frozen<Descendant[]>()),
+    lookOutSlateData: types.maybeNull(types.frozen<Descendant[]>()),
```

Comment:
We should refactor this to use quilljs which we can just store the raw html. If not handled in this PR then please make a separate card for it. We should eventually remove slate from the app

Reply:
I will do this as a part of the edit jurisdictiion card